engine/src/main/scala/cromwell/engine/workflow/ExecutionStoreKey.scala

```diff
@@ -1,47 +1,32 @@
 package cromwell.engine.workflow
 
+import cromwell.engine.ExecutionStatus
+import cromwell.engine.backend.JobKey
 import cromwell.engine.finalcall.FinalCall
-import wdl4s._
-import cromwell.engine.{WorkflowDescriptor, ExecutionStatus}
 import cromwell.engine.workflow.WorkflowActor.ExecutionStore
+import wdl4s._
 
 import scala.language.postfixOps
 
-sealed trait ExecutionStoreKey {
+sealed trait ExecutionStoreKey extends JobKey {
```

Comment:
Hmm this feels a little weird, the keys for the WA's internal execution store are extending a backend trait.  I probably shouldn't think about this too much.  :smile: